test/test_utils/sagemaker.py

```diff
@@ -200,7 +200,7 @@ def execute_local_tests(image, ec2_client):
     try:
         key_file = generate_ssh_keypair(ec2_client, ec2_key_name)
         print(f"Launching new Instance for image: {image}")
-        instance_id, ip_address = launch_sagemaker_local_ec2_instance(image, UBUNTU_16_BASE_DLAMI, ec2_key_name, region)
+        instance_id, ip_address = launch_sagemaker_local_ec2_instance(image, UBUNTU_16_BASE_DLAMI_US_WEST_2, ec2_key_name, region)
```

Comment:
can we conditionally set this AMI? I see we are getting region from AWS_REGION in line 197, we can use that to set the correct AMI

Reply:
We could. I'll make the change.